NetKVM/Common/ParaNdis_TX.cpp

```diff
@@ -755,6 +755,14 @@ bool CParaNdisTX::DoPendingTasks(CNBL *nblHolder)
 void CNB::MappingDone(PSCATTER_GATHER_LIST SGL)
 {
     m_SGL = SGL;
+#if NDIS_SUPPORT_NDIS650
+    #if ALICLOUD_MAX_FRAGMENTS_IN_ONE_NB
+        if (SGL->NumberOfElements > m_Context->uMaxFragments)
+        {
+            AllocateAndFillCopySGL();
```

Comment:
AllocateAndFillCopySGL may fail, in this case the driver will try to send the original packet with too many fragments. Some hardware may drop such packet, some hardware may enter erroneous state when such it gets such packet. Shouldn't we return 'sending fail' for such a packet?